stdlib/2and3/dummy_threading.pyi

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
this module (and _dummy_threading) is removed in python3.9 -- does typeshed have a way to represent that?

Reply:
No, see #16